### ISSUE TITLE
Concurrent Modification Exception

### ISSUE THREAD

**Issue by devryb** [gamerforEA/Cauldron-Reloaded_CLOSED#19]
Anyone else getting this crash alot?  We seem to get it quite a bit and it isn't really specific to any certain Cauldron build or modpack.
The only thing I have seen it possibly being is chunk loading issues with Ender Quarries.
Yes resetting the affected world helps for a bit but the issue returns as soon as players start quarrying out the Nether again.

We have been trying to find a solution for the better part of a month without luck and I'm just wondering at this point if anyone else has seen a similar problem

There's a few different crash reports that are all about the same thing, the server logs do not show anything in them before the crashes.

http://pastebin.com/tCxF1bky
http://pastebin.com/881g5dKb
http://pastebin.com/MXXrHj4V

**Comment by Myrninvollo** [comment]
Hi Q have you tryed removing the Ender Qurries mod and see if you still have the same issues or errors ?

**Comment by devryb** [comment]
I have not, removing Extra Utilities mod on the server would be pretty detrimental to the players and unfortunately I don't know exactly how to reproduce it so testing on my own wouldn't work out well.

**Comment by TheBrain0110** [comment]
The usual first question: is this crash specific to Cauldron, or does it happen on just regular Forge too?

**Comment by devryb** [comment]
Yep, again I can't really remove that on a live server and don't know how to reproduce on my own.  This was just a general query to other server owners if they have seen the same thing.

I guess the only way to proceed then is setup 50 ender qarries in the nether on a test server and see what happens

**Comment by spannerman79** [comment]
You need to download and examine the world data, specifically this from the first paste;

    Level spawn location: World: (0,64,0), Chunk: (at 0,4,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)

Only you can do that to find out *exactly* what is causing the issue

**Comment by devryb** [comment]
Hmm i don't seen any issues with the spawnpoint?  As far as I can tell the crash report isn't for a specific block in world issue

**Comment by devryb** [comment]
I will try to do some extensive testing with ender quarries on my own to see if I can reproduce it

**Comment by spannerman79** [comment]
Well it seems you must be new at managing anything to do with minecraft as you would know that it won't state a set co-ords 
https://www.youtube.com/watch?v=fWUUsdp3pJ8&hd=1

**Comment by devryb** [comment]
Your video linked begs otherwise....

**Comment by TheBrain0110** [comment]
I have had similar looking crashes before, I think Thermal Dynamics, EnderIO and Extra Utilities have all been culprits ticking either quarries or conduits/pipes, but they've all been bugs on their end, not Cauldron, and solved by updating the mods. I assume you have everything up to date?

**Comment by devryb** [comment]
Well as far as Direwolf 1.3.1 and Infinity 1.5.1 goes yes. I am currently restoring a copy of a map from the server most affected by this and will test it out on my own. If I can reproduce it i'll start eliminating cauldron etc.

**Comment by jburrows82** [comment]
Run your pack on vanilla forge, and link the pastebin crashlog. Let's eliminate Cauldron from the equation and then if it still crashes, you can go to the pack maker. FTB does not support Cauldron btw.

**Comment by devryb** [comment]
Managed to reproduce this crash on a test server using the backup copy from just before 20 crashes of this happened a couple days ago.
Found that running with cauldron causes the crash after about 15 minutes the server was up
Tried with forge and let it run for 30 minutes without a crash
I then changed load-chunk-on-request to false in cauldron.yml and ran the server with cauldron for 30 minutes without a crash on the same map copy from the same point in time.

I believe setting this will greatly reduce these crashes if not eliminate them but we will see.
We're currently going over every server we have and ensuring that setting is set to false
